lib/TemplatedPathPlugin.js

```diff
@@ -7,6 +7,7 @@
 const REGEXP_HASH = /\[hash(?::(\d+))?\]/gi,
 	REGEXP_CHUNKHASH = /\[chunkhash(?::(\d+))?\]/gi,
 	REGEXP_MODULEHASH = /\[modulehash(?::(\d+))?\]/gi,
+	REGEXP_CONTENTHASH = /\[contenthash(?::(\d+))?\]/gi,
```

Comment:
Does `loader-utils` also need support for this; or does it just forward the call to webpack?

https://github.com/webpack/loader-utils#interpolatename

Reply:
@Kovensky :+1:

Reply:
This is unrelated to getName in loader-utils